jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -1,5 +1,10 @@
-def step000_provision_sut() {
-    echo "Hi from new step000 method! TODO: I should be provisioning your SUT, but I'm not."
+def step000_provision_sut(SKIP_PROVISIONING, script_dir) {
+    echo "SKIP PROVISIONING?: ${SKIP_PROVISIONING} (${SKIP_PROVISIONING.class})"
```

Comment:
Yeah.  I had that on the other parameter for skipping PE install too.  Might remove these debug statements at some point.